staging/src/k8s.io/apimachinery/pkg/runtime/interfaces.go

```diff
@@ -210,6 +210,18 @@ type ObjectCreater interface {
 	New(kind schema.GroupVersionKind) (out Object, err error)
 }
 
+// ResourceMapper provides information about equivalent resources,
+// and the kind associated with a given resource
+type ResourceMapper interface {
+	// ResourcesFor returns a list of resources equivalent to resource.
+	// If subresource is specified, only equivalent resources which have the same subresource are returned.
+	ResourcesFor(resource schema.GroupVersionResource, subresource string) []schema.GroupVersionResource
```

Comment:
`EquivalentResourcesFor`. `ResourcesFor` is underspecified.

Reply:
done